semantic_conventions/trace/database.yaml

```diff
@@ -382,6 +382,49 @@ groups:
           The collection being accessed within the database stated in `db.name`.
         examples: [ 'customers', 'products' ]
 
+  - id: db.elasticsearch
+    prefix: db.elasticsearch
+    type: span
+    extends: db
+    brief: >
+      Call-level attributes for Elasticsearch
+    attributes:
+      - id: doc_id
+        type: string
+        requirement_level:
+          conditionally_required: when the request targets a specific document by id
+        tag: call-level-tech-specific-elasticsearch
+        brief: >
+          The document that the request targets.
+        examples: [ '123', '456' ]
+      - id: target
+        type: string
+        requirement_level:
+          conditionally_required: when a specific index or data stream is targeted by the request
+        tag: call-level-tech-specific
+        brief: >
+          The name of the data stream or index that is targeted.
+        examples: [ 'users' ]
+      - ref: url.query
+        requirement_level:
+          conditionally_required: when query params are provided as part of the request
+        brief: >
+          The query params of the request, as a json string.
+        examples: [ '"{\"q\":\"test\"}", "{\"refresh\":true}"' ]
+      - ref: url.path
```

Comment:
HTTP client spans have `url.full`, but not the individual components, so would it make sense to populate it? or populate individual parts specific to elastic like `target`, `doc_id`? or both?

Reply:
I'm not sure if I understand what you are proposing. Can you clarify if you are proposing not to have `url.path` but instead `url.full`? and whether you are proposing to keep the individual components as span attributes?

I think there is some value to having the individual parts as span attributes if, for example, you want to visualize spans that query particular indexes in Elasticsearch. The name of the span should be the path containing placeholders for these individual values, so I think there should be another way to target spans based on the placeholder values.